### PR TITLE
60 update frontend readme

### LINKED ISSUE #60: Update frontend readme
Update SAT frontend readme using the [gateway template](https://github.com/AmplicaLabs/gateway/blob/main/templates/README.template..md).

### PR DESCRIPTION
**Pull request by claireolmstead**
# Purpose
The goal of this PR is update the frontend readme using the template

Closes #60 

## Solution

Updated the readme using the template

### Change summary
* Took some content from the top level readme and combined it with the content from the current readme to fill out the template.

## Steps to Verify
1. Read through and make sure it all sounds right.
2. Make sure there are no back links.

### CODE REVIEW

File: frontend/README.md [hunk 1]
```diff
@@ -1,8 +1,219 @@
-# Getting Started with Create React App
+# Social App Template Frontend
+
+<!--
+Find and replace list:
+
+- `{gateway-service-repo}`: Example: `account-service`, `graph-service`
+- `{gateway-service-title}`: Example `Account Service`, `Graph Service`
+-->
```
Comment (claireolmstead) [inline frontend/README.md:8]:
```suggestion
```

**Review by JoeCap08055 — APPROVED** [review]
🚢 it!

File: frontend/README.md [hunk 2]
```diff
@@ -1,8 +1,212 @@
...
+<!-- Features -->
+
+### Key Features
+
+- **[Sign in with Frequency](https://github.com/AmplicaLabs/siwf)**
+- **Create a post**
+- **Create a comment on a post**
+- **View a feed**
+- **View other users' profiles**
+- **Follow a user**
```
Comment (mattheworris) [inline frontend/README.md:173]:
```suggestion
- **Follow/Unfollow a user**
```

**Review by mattheworris — APPROVED** [review]
Nice!

🚢 it!

**Pull request merged**